src/hooks/useAcceleratorConsole.ts

```diff
@@ -1,12 +1,13 @@
-import { useMatch } from 'react-router-dom';
+import { useLocation } from 'react-router-dom';
 
 import { APP_PATHS } from 'src/constants';
 import { useUser } from 'src/providers';
 import { isAllowed } from 'src/utils/acl';
 
 const useAcceleratorConsole = () => {
-  const isAcceleratorRoute = !!useMatch(APP_PATHS.ACCELERATOR);
   const { user } = useUser();
+  const location = useLocation();
+  const isAcceleratorRoute = location.pathname.startsWith(APP_PATHS.ACCELERATOR);
```

Comment:
I just checked and we can do:

`  const isAcceleratorRoute = !!useMatch({ path: APP_PATHS.ACCELERATOR, end: false });`

Reply:
Nice, let me try that 👍

Reply:
That worked, updating PR.